src/journal/views.py

```diff
@@ -433,6 +433,14 @@ def article(request, identifier_type, identifier):
     if article_object.is_published:
         store_article_access(request, article_object, 'view')
 
+    if request.journal.disable_html_downloads:
+        # exclude any HTML galleys.
+        galleys = galleys.exclude(
+            file__mime_type='text/html',
+            type='html',
+            label='HTML'
+        )
+
```

Comment:
The `label` field is free text, so if somebody types something else in for an HTML galley, it might not match `HTML` exactly. Can we avoid filtering on label here?